sdk/ai/Azure.AI.Inference/src/Customized/AzureAIInferenceEmbeddingGenerator.cs

```diff
@@ -0,0 +1,187 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.Buffers;
+using System.Buffers.Binary;
+using System.Buffers.Text;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.AI.Inference;
+
+namespace Microsoft.Extensions.AI;
```

Comment:
The rest of Inference uses the older form of namespace declarations.   We should convert those or these so that there's consistency in the package.

Reply:
Good catch; thanks. I'll switch to block-scoped namespaces so the PR's changes stay limited to what's being added/updated.